app-quizz/src/app/utils/renderCategories.js

```diff
@@ -0,0 +1,56 @@
+import React, {useState} from "react";
+
+export default function renderCategories() {
+  const [inputs, setInputs] = useState([""]);
+
+  const addInput = () => {
+    setInputs(["", ...inputs]);
+  };
+
+  const addCategoryIcon = (
+    <div>
+      <svg
```

Comment:
S: Perhaps it would be advantageous to extract this SVG and place it in a separate file.